Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -92,6 +94,43 @@ using SMatrix55Sym = ROOT::Math::SMatrix<double, 5, 5, ROOT::Math::MatRepSym<dou
 namespace o2::aodproducer
 {
 
+void AODProducerWorkflowDPL::createCTPReadout(const o2::globaltracking::RecoContainer& recoData, std::vector<o2::ctp::CTPDigit>& ctpDigits, int runNumber)
+{
+  // Extraxt CTP Config from CCDB
+  o2::ctp::CTPConfiguration ctpcfg = o2::ctp::CTPRunManager::getConfigFromCCDB(-1, std::to_string(runNumber)); // how to get run
```

Comment:
getConfigFromCCDB still uses BasicCCDBManager which we are eliminating from the production workflows. Instead of using this method, use DPL CCDB functionality, see e.g. https://docs.google.com/presentation/d/1WPNixo57zAn_gM6vMCIBOHs-DZqcKAkLX0CwUgO_VYg/edit#slide=id.g10e79f44b99_0_7